### DISCUSSION TITLE
Flag to show end of file

### DISCUSSION THREAD

**Discussion by Venomish-Snake** [BurntSushi/ripgrep#2514] [General] [answered]
Hello!
I have recently experienced the situation below and used a flag to find the end of file but cannot remember the flag(s) used.

I have a file in UTF-8 encoding, and try searching a string with the command "rga -i "carsbootpills"". Although the string is present in the file, it does not show that it is matched. 
As per above, i had the same issue and used a flag which showed that there was a nullbyte on byte x. I solved the issue by removing that nullbyte.

If anyone can remind me of the correct flag to use to show the end of file, please let me know!

Thanks!!

**Comment by BurntSushi** [comment, marked as answer]
I don't understand the question. I don't know what you mean by "show end of file."

If you want to search binary data, then use the `-a/--text` flag.
